{{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/ext.py

```diff
@@ -28,7 +29,6 @@ def init_app(self, app):
 
     def init_config(self, app):
         """Initialize configuration."""
-        app.config.setdefault(
-            "{{ cookiecutter.config_prefix}}_BASE_TEMPLATE",
-            app.config.get("BASE_TEMPLATE",
```

Comment:
This won't work correctly.  ``<MODULE>_BASE_TEMPLATE`` need to be set dynamically from ``app.config['BASE_TEMPLATE']`` (see https://invenio-theme.readthedocs.io/en/latest/usage.html#base-template-pattern)

```python
# config.py
{{ cookiecutter.config_prefix}}_BASE_TEMPLATE = None
# ext.py